src/App.tsx

```diff
@@ -17,18 +68,33 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filterType={filterType}
+                query={query}
+                setFilterType={setFilterType}
+                setQuery={setQuery}
```

Comment:
Read the article from checklist about naming props and functions and apply the changes in all places